.travis.yml

```diff
@@ -8,6 +8,7 @@ before_install:
 - export CHROME_BIN=chromium-browser
 - export DISPLAY=:99.0
 - sh -e /etc/init.d/xvfb start
+script: grunt test && grunt coveralls
```

Comment:
Why not just keep the default and have `npm test` either get passed wither to run coveralls as an option or figure it out itself?